holoviews/tests/core/test_utils.py

```diff
@@ -182,180 +185,163 @@ def test_deephash_nested_mixed_inequality(self):
 
     def test_underscore_string(self):
-        self.assertEqual(tree_attribute('_underscore'), False)
+        assert not tree_attribute('_underscore')
 
 
 
-class TestSanitization(ComparisonTestCase):
+class TestSanitization:
     """
     Tests of sanitize_identifier
     """
     def test_simple_pound_sanitized(self):
         sanitized = sanitize_identifier('£')
-        self.assertEqual(sanitized, 'pound')
+        assert sanitized == 'pound'
 
     def test_simple_digit_sanitized(self):
         sanitized = sanitize_identifier('0')
-        self.assertEqual(sanitized, 'A_0')
+        assert sanitized == 'A_0'
 
     def test_simple_underscore_sanitized(self):
         sanitized = sanitize_identifier('_test')
-        self.assertEqual(sanitized, 'A__test')
+        assert sanitized == 'A__test'
 
     def test_simple_alpha_sanitized(self):
         sanitized = sanitize_identifier('α')
-        self.assertEqual(sanitized, 'α')
+        assert sanitized == 'α'
 
     def test_simple_a_pound_sanitized(self):
         sanitized = sanitize_identifier('a £')
-        self.assertEqual(sanitized, 'A_pound')
+        assert sanitized == 'A_pound'
 
     def test_capital_delta_sanitized(self):
         sanitized = sanitize_identifier('Δ')
-        self.assertEqual(sanitized, 'Δ')
+        assert sanitized == 'Δ'
 
     def test_lowercase_delta_sanitized(self):
         sanitized = sanitize_identifier('δ')
-        self.assertEqual(sanitized, 'δ')
+        assert sanitized == 'δ'
 
     def test_simple_alpha_beta_sanitized(self):
         sanitized = sanitize_identifier('α β')
-        self.assertEqual(sanitized, 'α_β')
+        assert sanitized == 'α_β'
 
     def test_simple_alpha_beta_underscore_sanitized(self):
         sanitized = sanitize_identifier('α_β')
-        self.assertEqual(sanitized, 'α_β')
+        assert sanitized == 'α_β'
 
     def test_simple_alpha_beta_double_underscore_sanitized(self):
         sanitized = sanitize_identifier('α__β')
-        self.assertEqual(sanitized, 'α__β')
+        assert sanitized == 'α__β'
 
     def test_simple_alpha_beta_mixed_underscore_space_sanitized(self):
         sanitized = sanitize_identifier('α__  β')
-        self.assertEqual(sanitized, 'α__β')
+        assert sanitized == 'α__β'
 
     def test_alpha_times_two(self):
         sanitized = sanitize_identifier('α*2')
-        self.assertEqual(sanitized,  'α_times_2')
-
-    def test_arabic_five_sanitized(self):
-        """
-        Note: There would be a clash if you mixed the languages of
-        your digits! E.g. arabic ٥ five and urdu ۵ five
-        """
-        try:
-            sanitize_identifier('٥')
-        except SyntaxError as e:
-            assert str(e).startswith("String '٥' cannot be sanitized")
-
-    def test_urdu_five_sanitized(self):
-        try:
-            sanitize_identifier('۵')
-        except SyntaxError as e:
-            assert str(e).startswith("String '۵' cannot be sanitized")
```

Comment:
These tests do not raise a syntax error, not sure if they ever failed. Also checked on Windows.